externs/shaka/player.js

```diff
@@ -1400,6 +1401,14 @@ shaka.extern.MssManifestConfiguration;
  *   the video is paused.
  *   <br>
  *   Defaults to <code>true</code>.
+ * @property {boolean} overrideDolbyVisionCodecs
+ *   If true, Dolby Vision codecs in a manifest will be overriden to their
+ *   AVC/HEVC equivalents. This is a workaround for platforms which MediaSource
+ *   methods claim they are not able to play Dolby Vision, but when segments are
+ *   appended it appears to play ocrrectly.
+ *   <br>
+ *   Defaults to <code>true</code> on Xbox One and <code>false</code> on other
```

Comment:
Design-wise, is there a reason why a developer would want to change this value from the default? I'm not sure if we are really expecting any other platforms to have this specific problem, besides Xbox One.

If this is a workaround that will probably always be left at the default value, it might be better if the stream utils just call `shaka.util.Platform.isXboxOne()` directly to determine if they should change the codecs, instead of passing it in as a config value. We already have platform-specific encoding logic for Xbox One (and android Firefox) inside `shaka.util.StreamUtils.checkVariantSupported_`, so it'd hardly be anything new.

Obviously if you do this, the other two comments I made will no longer be relevant.

Reply:
I wanted to make it configurable in case there are other platforms that need that fix in the future. But right, I don't know about any other, so let's make it always forced on Xbox One.